bot.py

```diff
@@ -83,7 +83,7 @@ def getTownInfo(self, ctx):
         doc = guildInfo.find_one(query)
 
         if doc:
-            return TownInfo(ctx, doc)
+            return TownInfo(ctx.guild, doc)
```

Comment:
This is gonna fail - ctx doesn't exist

Reply:
I don't understand - ctx was what was previously being passed here, now it's ctx.guild

Reply:
It's possible the phone app is not showing me everything - I'll have to check later on my PC.

Reply:
I missed something on phone - I thought this was the body of the func above where ctx had been removed in favor of guild 😆 didn’t notice the “x lines collapsed” thingy